### PR TITLE
Correct url for Karma website

### PR DESCRIPTION
**Pull request by MortenGregersen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,5 +118,5 @@ module.exports = function(config) {
 For more information on Karma see the [homepage].
 
 
-[homepage]: http://karma-runner.github.com
+[homepage]: http://karma-runner.github.io
```
Comment (anthony-redFox) [inline README.md:121]: Will be better use https

Comment (MortenGregersen) [reply README.md:121]:
```suggestion
[homepage]: https://karma-runner.github.io
```

**Review by anthony-redFox — CHANGES_REQUESTED** [review]
please check your commits on https://www.conventionalcommits.org/en/v1.0.0/

**Comment by MortenGregersen** [comment]
> please check your commits on https://www.conventionalcommits.org/en/v1.0.0/

Better now?

**Comment by MortenGregersen** [comment]
I can't reopen the pull request.

**Pull request closed without merging**